NuGet.Docs/Create/TargetFrameworks.md

```diff
@@ -0,0 +1,647 @@
+## Target Frameworks
+
+As the .NET ecosystem has grown, so too has the list of frameworks and dependencies that NuGet supports.   The frameworks that can be included and referenced by a NuGet package tracks closely with the list of available .NET framework versions and flavors that support all of the devices and systems the .NET framework can be used on.
+
+Target frameworks can be referenced from three places:  nuspec manifest, nupkg folder name, and project.json frameworks node.  A framework is typically referenced by a short name, called a "Target Framework Moniker" or TFM.  With the advent of the new Platform Standard, the concept of 'TFM' has been abstracted to 'TxM' since the Platform Standard references multiple frameworks abstractly. 
```

Comment:
I'd be more specific about this. Explain how they manifest in the nuspec (like on the dependencies element) and in project.json.

Also explain you can have assets that vary by TFM (certain ones).